packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -3566,7 +3581,8 @@ export class ThinEngine extends AbstractEngine {
             texture: InternalTexture,
             continuationCallback: () => void
         ) => boolean,
-        samplingMode: number = Constants.TEXTURE_TRILINEAR_SAMPLINGMODE
+        samplingMode: number,

```

Comment:
`Constants.TEXTURE_TRILINEAR_SAMPLINGMODE` is already specified as the default samplingMode in AbstractEngine, so this one is redundant. Removed to simplify and make it easy to add one more param at the end.